### DISCUSSION TITLE
Conversion from Nemo to tensorflow model

### DISCUSSION THREAD

**Discussion by mjhanphd** [NVIDIA-NeMo/NeMo#6460] [Q&A]
Hi, has anybody ever tried to convert a nemo model to Tensorflow model?
What I really need is to pretrain on Nemo platform and convert the pretrained model (e.g. xxxx.nemo) to Tensorflow ckpt and finetune under TF frameworks.
After some digging, I found that I can convert xxxx.nemo -> xxxx.onnx -> xxxx.pb where pb is Google's protobuf format containing the model graph and weights.
However being able to train on TF is different since for doing that I need to implement the model layers in Tensorflow/Keras grammars.
I'm wondering if there's anybody with even a little experience on this.
Thank you.

**Comment by N-yokoya** [comment]
Hello! I have no experience for your discussion. But I ask your question to OPEN_AI and get an answer. Do you want to know it? Or you can try it by yourself. Best regards.
